style.css

```diff
@@ -441,5 +470,486 @@ textarea::placeholder {
+    width: 75.87%;
+    font-size: 2.5rem;
+    font-weight: 700;
+    line-height: 3.25rem;
+    text-align: left;
+    margin-left: 24.1%;
+    margin-bottom: 1.31%;
+    margin-top: 15.5%;
+    margin-right: 0;
+  }
+
+  #about-m-pag-desktop {
+    height: auto;
+    width: 76%;
+    font-size: 1rem;
+    font-weight: 400;
+    color: #344563;
+    margin-left: 24.1%;
+    margin-bottom: 1.31%;
+    font-family: 'Poppins', sans-serif;
+    text-align: left;
+  }
+
+  #about-m-letsconnect-desktop {
+    margin-left: 24.1%;
+    margin-bottom: 2.838%;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1rem;
+    line-height: 1.5rem;
+    color: #7f8cff;
+  }
+
+  .see-project-button {
+    margin-top: 4.366%;
+  }
+
+  #about-m-media-links-desktop-margin {
+    margin-left: 24.1%;
+    margin-top: 2.5%;
+  }
+
+  #about-me-desktop-button-margin {
+    margin-left: 24.1%;
+  }
+
+  #about-me-desktop-lang-margin {
+    margin-top: 1.812%;
+  }
+
+  .dev-language {
+    flex-direction: column;
+    display: flex;
+    margin-right: 1.662%;
+    align-items: flex-start;
+    width: 14.9%;
+  }
+
+  .margin-lang-frame {
+    margin-right: 12.7423%;
+  }
+
+  .image-lang {
+    padding-left: 0;
+  }
+
+  .desktop-lang-font {
+    height: 2.62%;
+    font-size: 1.25rem;
+    font-weight: 500;
+    line-height: 1.5rem;
+    text-align: left;
+    color: #344563;
+  }
+
+  .lang-span {
+    margin-left: 0;
+    width: 6.7867%;
+  }
+
+  .contact-me-background {
+    position: relative;
+    background-image: url(Images/BG\ end\ Contact\ Form\ Desktop.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  .header-contact-me {
+    margin-top: 15.5%;
+    margin-bottom: 1.31%;
+  }
+
+  .contact-me-paragraph {
+    line-height: 1.75rem;
+    padding-bottom: 1rem;
+    width: 39.5138%;
+    height: 11.135%;
+    margin-left: 30.19%;
+    margin-right: 30.19%;
+    margin-bottom: 1.31%;
+    font-size: 1.25rem;
+  }
+
+  #contact-me-form {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .contact-me-name {
+    height: 5.24%;
+    width: 30.95567%;
+    border-radius: 0.125rem;
+    margin-bottom: 2.957%;
+    text-align: left;
+  }
+
+  .contact-me-email {
+    height: 5.24%;
+    width: 30.95567%;
+    margin-bottom: 2.94759%;
+    border-radius: 0.125rem;
+  }
+
+  .contact-me-textarea {
+    height: 18.5589%;
+    width: 30.95567%;
+    margin-bottom: 2.94759%;
+    border-radius: 0.125rem;
+  }
+
+  #contact-me-submit-button-desktop {
+    margin-bottom: 13.864%;
+    border-radius: 0.5rem;
+    height: 5.24%;
+    margin-left: 0;
+    padding: 0.75rem;
+    border: 0.0625rem solid #396df2;
+    color: #396df2;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1.0625rem;
+    background-color: #fff;
+  }
+
+  #margin-desktop-language {
+    margin-top: 18.12%;
+  }
```

Comment:
- Hi @stenlisuryadinata :wave: , I understand how you may feel now, and would really like to commend your efforts on this task. You are just very close :hugs: 
- [ ] In [this previous review](https://github.com/HFG43/Portfolio-Mobile-Version/pull/6#discussion_r1125369783) by a colleague, he only asked that you use relative units like `percentages` for your styling so that you can remove the overflow triggering a horizontal scrollbar on your page- which is not so good for user experience. It appears that while you were at it, a lot of things have been broken into and your some of your good designs have been affected. I have attached a picture of then and now below so that you can adjust the card-images that are stretching beyond their containers - (I'll recommend a `max-width: 100%;` on your images).

|Current Design| Previous Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/222926920-3e3b065a-c145-42e6-98f8-20057dc36542.png)|![image](https://user-images.githubusercontent.com/99715640/222926938-a51bcbcc-99d6-4375-9f9b-7171f019b482.png)|

- _**Kindly address the scrollbar issue and other broken parts due to broken images, fixed (absolute) units for height declaration.**_